data/ui/oops-window.ui

```diff
@@ -1,257 +1,98 @@
-        <property name="related_action">gac_report</property>
-        <property name="visible">True</property>
-        <property name="can_focus">False</property>
-      </object>
-    </child>
-    <child>
-      <object class="GtkMenuItem" id="mi_delete">
-        <property name="tooltip_text" translatable="yes">Delete selected problems</property>
-        <property name="related_action">gac_delete</property>
-        <property name="visible">True</property>
-        <property name="can_focus">False</property>
-      </object>
-    </child>
-    <child>
-      <object class="GtkMenuItem" id="mi_detail">
-        <property name="tooltip_text" translatable="yes">Show technical details</property>
-        <property name="related_action">gac_detail</property>
-        <property name="visible">True</property>
-        <property name="can_focus">False</property>
-      </object>
-    </child>
-    <child>
-      <object class="GtkMenuItem" id="mi_open_directory">
-        <property name="tooltip_text" translatable="yes">Opens the selected problem's data directory in the default file system browser</property>
-        <property name="related_action">gac_open_directory</property>
-        <property name="visible">True</property>
-        <property name="can_focus">False</property>
-      </object>
-    </child>
-    <child>
-      <object class="GtkMenuItem" id="mi_copy_id">
-        <property name="tooltip_text" translatable="yes">Stores the selected problem's ID in Clibpoard</property>
-        <property name="related_action">gac_copy_id</property>
-        <property name="visible">True</property>
-        <property name="can_focus">False</property>
-      </object>
-    </child>
-    <child>
-      <object class="GtkMenuItem" id="mi_search">
-        <property name="tooltip_text" translatable="yes">Show filter box</property>
-        <property name="related_action">gac_search</property>
-        <property name="can_focus">False</property>
-      </object>
-    </child>
-  </object>
-  <object class="GtkImage" id="image1">
-    <property name="visible">True</property>
-    <property name="can_focus">False</property>
-    <property name="icon_name">object-select-symbolic</property>
-  </object>
-  <object class="GtkImage" id="image2">
-    <property name="visible">True</property>
-    <property name="can_focus">False</property>
-    <property name="icon_name">edit-find-symbolic</property>
-  </object>
-  <object class="GtkWindow" id="wnd_main">
+  <template class="OopsWindow" parent="GtkApplicationWindow">
     <property name="width_request">1024</property>
     <property name="height_request">768</property>
     <property name="can_focus">False</property>
     <property name="icon_name">gnome-abrt</property>
-    <accel-groups>
-      <group name="ag_accelerators"/>
-    </accel-groups>
-    <child>
-      <object class="GtkBox" id="box_window">
+    <child type="titlebar">
+      <object class="GtkHeaderBar" id="header_bar">
```

Comment:
(I'm not testing/building the code, so idk) Does this changes between SSD and CSD?

Reply:
Nope, the header bar was being created in the code before.